### PR TITLE
Rails6.1

### PR DESCRIPTION
**Pull request by doyle**

### CODE REVIEW

File: lib/activeuuid/patches.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 require 'active_record'
 require 'active_support/concern'
 
-if ActiveRecord::VERSION::MAJOR == 4 and ActiveRecord::VERSION::MINOR == 2 or ActiveRecord::VERSION::MAJOR == 5
+if ActiveRecord::VERSION::MAJOR == 4 and ActiveRecord::VERSION::MINOR == 2 or ActiveRecord::VERSION::MAJOR >= 5
```
Comment (cameronf) [inline lib/activeuuid/patches.rb:4]: if we are binding to activerecord 6+ why do we need all of these conditionals?

File: lib/activeuuid/patches.rb [hunk 2]
```diff
@@ -172,10 +172,10 @@ module Quoting
         # alias_method_chain :native_database_types, :uuid
       # end
         def quote(value)
-          if value && value.try(:type).is_a?(ActiveRecord::Type::UUID)
-            return "'#{value.value}'" unless value.value.respond_to?(:raw)
```
Comment (eszarpa) [inline lib/activeuuid/patches.rb:176]: Ugh this stuff changed AGAIN? :(

**Review by jas13 — APPROVED** [review]
Ummm not to be "that" guy, but why did we break this off from `activeuuid` in the first place? Also this is a fork about 4 forks deep.

**Comment by doyle** [comment]
@jas13 the original hasn't been updated since 2015, we forked from the most up to date fork

File: lib/activeuuid/patches.rb (hunk 2 above)
Comment (doyle) [reply lib/activeuuid/patches.rb:176]: Yeah everything was fine in 6, but broke again in 6.1

**Comment by jas13** [comment]
@eszarpa @doyle Maybe we should consider a tech debt ticket to switch to https://www.ruby-toolbox.com/projects/mysql-binuuid-rails

Which appears to be maintained and widely used.

**Pull request merged**